src/addon.cpp

```diff
@@ -437,7 +504,7 @@ NAN_MODULE_INIT(init)
   JS_CL_CONSTANT(Rx);
   JS_CL_CONSTANT(RGx);
   JS_CL_CONSTANT(RGBx);
-#ifdef  CL_VERSION_1_2
+#ifdef  v12
```

Comment:
There are couple of things I wanted to address in relation to macros, first that macros shouldn't take responsibility and turn off deprecated OpenCL features just because they are deprecated, if there's a feature and works it should be available through the binding